cves/kernel/CVE-2021-29647.yml

```diff
@@ -106,9 +101,9 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: bdabad3e363d825ddf9679dd431cca0b2c30f881
-  :note: Discovered automatically by archeogit.
+  :note: It is probable the vulnerability was introduced here
 - commit: f507a9b6e63b9f41b61d199c36ae046d17b5fe4b
-  :note: Discovered automatically by archeogit.
+  :note: It is not probable that the vulnerability was introduced here, but this commit is more than just refactoring
 upvotes_instructions: |
```

Comment:
I would give this 5 of my upvotes.

Reply:
I meant to go back after I had a better understanding of the defect to write a better note there

Reply:
I would give this 2 of my upvotes